ui/app/AppLayouts/Wallet/services/dapps/DAppsListProvider.qml

```diff
@@ -18,7 +18,12 @@ QObject {
     readonly property int connectorId: Constants.WalletConnect
     readonly property var dappsModel: d.dappsModel
 
+    property bool enabled: true
+
     Component.onCompleted: {
+        if (!enabled) {
+            return
+        }
         // Just in case the SDK is already initialized
         d.updateDappsModel()
```

Comment:
shouldn't it be done on enabled changed to true?

Reply:
yes, that's true! Thank you 🍻 
Will update